exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/QaServiceImplIntegrationTest.java

```diff
@@ -223,10 +224,8 @@ void submitUnknownTx() throws Exception {
 
     HashCode txHash = service.submitUnknownTx();
 
-    Transaction expectedTx = new UnknownTx();
-
-    assertThat(txHash).isEqualTo(expectedTx.hash());
-    verify(node).submitTransaction(any(UnknownTx.class));
+    assertThat(txHash).isNotNull();
+    verify(node).submitTransaction(any(RawTransaction.class));
```

Comment:
This test doesn't test much.

Reply:
added a random seed to the transaction. that's because such assert